<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Michelles-MacBook-Pro-2.local" ignored="0" name="Default test" tests="8" failures="1" timestamp="12 Feb 2019 01:58:16 GMT" time="17.672" errors="0">
  <testcase name="InternalServerErrorCase" time="0.282" classname="Dais_Technologies.Code_Exercise_paint_calculator_AutomationScript.BaseTest">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 
at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
at org.testng.AssertJUnit.assertTrue(AssertJUnit.java:24)
at org.testng.AssertJUnit.assertFalse(AssertJUnit.java:41)
at org.testng.AssertJUnit.assertFalse(AssertJUnit.java:49)
at Dais_Technologies.Code_Exercise_paint_calculator_AutomationScript.BaseTest.InternalServerErrorCase(BaseTest.java:80)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- InternalServerErrorCase -->
  <testcase name="Redirect" time="0.224" classname="Dais_Technologies.Code_Exercise_paint_calculator_AutomationScript.BaseTest"/>
  <testcase name="TextboxLength" time="0.03" classname="Dais_Technologies.Code_Exercise_paint_calculator_AutomationScript.BaseTest"/>
  <testcase name="TextboxVerify_Decimal" time="0.129" classname="Dais_Technologies.Code_Exercise_paint_calculator_AutomationScript.BaseTest"/>
  <testcase name="TextboxVerify_Negative" time="0.123" classname="Dais_Technologies.Code_Exercise_paint_calculator_AutomationScript.BaseTest"/>
  <testcase name="TextboxVerify_Zero" time="0.117" classname="Dais_Technologies.Code_Exercise_paint_calculator_AutomationScript.BaseTest"/>
  <testcase name="TextboxVerify_null" time="0.107" classname="Dais_Technologies.Code_Exercise_paint_calculator_AutomationScript.BaseTest"/>
  <testcase name="isElementpresent" time="0.145" classname="Dais_Technologies.Code_Exercise_paint_calculator_AutomationScript.BaseTest"/>
</testsuite> <!-- Default test -->
